use-timescale/hypertables/improve-query-performance.md

```diff
@@ -1,51 +1,156 @@
+
+## Optimize hypertable chunk intervals
+
+Adjusting your hypertable chunk interval can improve performance in your database.
+
+<Procedure>
+
+1. **Choose an optimum chunk interval**
+
+   The default chunk interval is 7 days. You can set a custom interval when you create a hypertable.
+   Best practice is that prior to processing, one chunk of data takes up 25% of main memory, including the indexes
+   from each active hypertable. For example, if you write approximately 2 GB of data per day to a database with 64
+   GB of memory, set `chunk_interval` to 1 week. If you write approximately 10 GB of data per day on the same
+   machine, set the time interval to 1 day. For more information, see [best practices for time partitioning][best-practices].
+
+   In the following example you create a table called `conditions` that stores time values in the
+   `time` column and has chunks that store data for a `chunk_interval` of one day:
+
+   ```sql
+   CREATE TABLE conditions (
+      time        TIMESTAMPTZ       NOT NULL,
+      location    TEXT              NOT NULL,
+      device      TEXT              NOT NULL,
+      temperature DOUBLE PRECISION  NULL,
+      humidity    DOUBLE PRECISION  NULL
+   ) WITH (
+      tsdb.hypertable,
+      tsdb.partition_column='time',
+      tsdb.chunk_interval='1 day'
+   );
+   ```
+
+   <OldCreateHypertable />
+
+1. **Check current setting for chunk intervals**
+
+   Query the $TIMESCALE_DB catalog for a $HYPERTABLE. For example:
+
+   ```sql
+   SELECT * 
+     FROM timescaledb_information.dimensions 
+     WHERE hypertable_name = 'conditions';
+   
+   ```
+
+   The result looks like:
+
+   ```sql
+   hypertable_schema | hypertable_name | dimension_number | column_name |       column_type        | dimension_type | time_interval | integer_interval | integer_now_func | num_partitions
+   -------------------+-----------------+------------------+-------------+--------------------------+----------------+---------------+------------------+------------------+----------------
+    public           | metrics          |                1 | recorded    | timestamp with time zone | Time           | 1 day         |                  |                  |
+   ```
+
+   Time-based interval lengths are reported in microseconds.
+
+1. **Change the chunk interval length on an existing hypertable**
+
+   To change the chunk interval on an already existing hypertable, call `set_chunk_time_interval`.
+
+   ```sql
+   SELECT set_chunk_time_interval('conditions', INTERVAL '24 hours');
+   ```
+
+   The updated chunk interval only applies to new chunks. This means setting an overly long
+   interval might take a long time to correct. For example, if you set
+   `chunk_interval` to 1 year and start inserting data, you can no longer
+   shorten the chunk for that year. If you need to correct this situation, create a
+   new hypertable and migrate your data.
+
+   // IAIN: I don't think this is true for 2.20 and o
```

Comment:
@philkra can you have a look at this please.

Reply:
This behavior did not change